executors.yaml

```diff
@@ -119,6 +119,11 @@ sh:
   commands:
     - ["sh", "$pwd/script.sh"]
   hidden_line_prefix: "/// "
+uv:
```

Comment:
Can you add this instead as an `alternative` executor for `python`? See https://github.com/mfontanini/presenterm/blob/master/executors.yaml#L103-L104. By doing this you can still use `python` as the language in your snippet (so you hopefully get highlighting in your editor) but you just have to specify how the program is ran:

~~~markdown
```python +exec:uv
print("hi")
```
~~~